stdlib/bytesLabels.mli

```diff
@@ -513,6 +513,8 @@ external unsafe_set : bytes -> int -> char -> unit = "%bytes_unsafe_set"
 external unsafe_blit :
   src:bytes -> src_pos:int -> dst:bytes -> dst_pos:int -> len:int ->
     unit = "caml_blit_bytes" [@@noalloc]
+external unsafe_blit_string : string -> int -> bytes -> int -> int -> unit
+                     = "caml_blit_string" [@@noalloc]
```

Comment:
This could usefully have `src_pos`, `dst_pos` and `len` labels to distinguish the various `int` arguments, as with `unsafe_blit` above.